gdpr/vendorlist-fetching_test.go

```diff
@@ -192,12 +193,16 @@ type versionInfo struct {
 }
 type saver []versionInfo
 
+var saverMutex sync.Mutex
+
 func (s *saver) saveVendorLists(specVersion uint16, listVersion uint16, gvl api.VendorList) {
 	vi := versionInfo{
 		specVersion: specVersion,
 		listVersion: listVersion,
 	}
+	saverMutex.Lock()
 	*s = append(*s, vi)
+	saverMutex.Unlock()
```

Comment:
If `	*s = append(*s, vi)` panics, I think that the `saverMutex` will never be unlocked.

I think that deferring can avoid that:

```golang
	saverMutex.Lock()
	defer saverMutex.Unlock()
	*s = append(*s, vi)
```

Reply:
updated

Reply:
Thanks